README.md

```diff
@@ -18,17 +18,28 @@ yarn add -D \
     prettier
 ```
 
-In your `eslint.config.cjs`:
+In your `eslint.config.mjs`:
 
 ```js
-const foxglove = require("@foxglove/eslint-plugin");
-const tseslint = require("typescript-eslint");
+import foxglove from "@foxglove/eslint-plugin";
+import tseslint from "typescript-eslint";
 
-module.exports = tseslint.config(
+export default tseslint.config(
   ...foxglove.configs.base,
   ...foxglove.configs.react,
```

Comment:
@jtbandes are some additional `files` overrides required for react and jest too?

Reply:
I’ve never noticed any problems from these rules running in places they shouldn’t, but maybe it’s possible? Not sure—did you run into any issues?